### ISSUE TITLE
Date Time Picker depends on the Dev release channel instead of the production

### ISSUE THREAD

**Issue by jamiethain** [danvick/flutter_form_builder#183]
Like many people our project does not pass QA with Dev Channel items it in. 

Can you change over to the release channel of date time picker.

Because flutter_form_builder 3.6.1 depends on datetime_picker_formfield ^1.0.0-pre.5 and no versions of flutter_form_builder match >3.6.1 <4.0.0, flutter_form_builder ^3.6.1 requires datetime_picker_formfield ^1.0.0-pre.5.

So, because flmobile depends on both datetime_picker_formfield ^0.4.3 and flutter_form_builder ^3.6.1, version solving failed.
pub get failed (1)
exit code 1

**Comment by danvick** [comment]
The reason why we did this is that the newer version addresses some issues that are not in the stable version. That's why we needed to do this.

The problem you are having is a dependency mismatch issue which is very common and unrelated.
Kindly have a look dependency overrides in Flutter/Dart, or refer to [this video](https://www.youtube.com/watch?v=nw2c6YI1Sb8) which shows how to deal with the issue.
